pyccel/codegen/printing/ccode.py

```diff
@@ -303,6 +303,15 @@ def _flatten_list(self, irregular_list):
             return [irregular_list]
 
     #========================== Numpy Elements ===============================#
+    def _largest_literal_subset(self, flattened_list):
```

Comment:
can you add doc string

Reply:
https://github.com/pyccel/pyccel/pull/1128/commits/d90fe3129b05b9974c51ee94de324654e4cb6bf3